src/App.scss

```diff
@@ -465,3 +465,9 @@
     text-align: left;
   }
 }
+
+.comparison-filter-input-dropdown-width {
```

Comment:
I am not sure if we really need a static width?

Reply:
I guess it's a matter of uniformity. If it don't set it, then the filter box size jumps around and I don't like the way it looks. Not sure what you think about this.